templates/default/client-auth-n.cfg.xml.erb

```diff
@@ -18,6 +18,13 @@
         <identity-service username="<%= @username %>" password="<%= @password %>" tenantId="<%= @tenant_id %>" uri="<%= @auth_uri %>" />
         <% if @tenanted %>
         <client-mapping id-regex="<%= @mapping_regex %>"/>
+        <% if not @ignore_tenant_roles.empty? %>
```

Comment:
Perhaps prefer `unless` over `if not`?  Its a chef/ruby style thing for negative condition matching.